ios/FluentUI/Button/Button.swift

```diff
@@ -181,21 +201,23 @@ open class Button: UIButton {
     }
 
     open override func imageRect(forContentRect contentRect: CGRect) -> CGRect {
```

Comment:
hmmm we need more docuemntatiopon and assert handling for ios 15? they shouldn't be calling this anymore right?

Reply:
Right, they shouldn't be calling it, but UIButton already has that. Are the warnings not inherited as well?

Reply:
we can makr this has @deprecated here if it is a override?

Reply:
Turns out trying to deprecate something the system calls makes it crash. Added an assert instead.